.gitattributes

```diff
@@ -4,6 +4,3 @@
 bin/* diff=ruby
 bundler/bin/* diff=ruby
 bundler/exe/* diff=ruby
-
-# Auto detect text files and perform LF normalization
-* text eol=lf
```

Comment:
I don't think if this was the real reason for the CI to break? :thinking:

Reply:
Yes, it is. But it's a mess because our CI checks out `rubygems` master branch, which doesn't have the updated `.gitattributes` file, so it's still broken :laughing:.

Reply:
Oh wow, then I think we're good to merge? 🔥

Reply:
I'm doing some experimentation with solutions that will allow us to actually catch these issues before merge, not after :sweat_smile:.

Reply:
It worked! Merging this to go back to green.